src/Web/HTML/Event/DataTransfer/DataTransferItem.purs

```diff
@@ -0,0 +1,35 @@
+module Web.HTML.Event.DataTransfer.DataTransferItem
+  ( DataTransferItem
+  , DataTransferItemList
+  ) where
+
+import Prelude
+import Data.Maybe (Maybe)
+import Data.Nullable (Nullable)
+import Data.Nullable as Nullable
+
+-- | Returns the drag data item kind, which is either "string" or "file".
+foreign import kind :: DataTransferItem -> String
```

Comment:
An alternative here would be to create a sum type for this return value, since the spec says that it must be one of either `"string"` or `"file"`. I don't know if that would be preferable.

Reply:
Perhaps something like `DataTransferItemType = StringItem | FileItem`? It's a bit ugly 🤷

Reply:
The spec calls them **Text** and **File** respectively. 
<img width="143" alt="image" src="https://user-images.githubusercontent.com/26548438/136483308-2e0bfbb0-2082-42b8-9685-eefc393fdbe6.png">
https://html.spec.whatwg.org/#dom-datatransferitem-kind

So my initial thought was
```haskell
data DataTransferItemKind = Text | File
```
but I can understand if those are too generic and we'd prefer to suffix with `*Kind` or `*Type` (or if `Text` is maybe an unexpected name for what has a JS value of `"string"`). Plenty of decisions here 😄 I'll push what I have but am open to suggestions for the naming.

Reply:
Hm, I've got this wrong actually... it can return an empty string "if the `DataTransferItem` object is in the disabled mode." I'll open this back up for review after I make the changes there.

Reply:
I could see suffixing them with `*Kind` to avoid clashes with other similar types, but I think it's also fine to assume this module will be imported qualified.